python/tank/api.py

```diff
@@ -221,6 +222,19 @@ def configuration_name(self):
     ##########################################################################################
     # public methods
 
+    def log_core_metric(self, action):
```

Comment:
Making it not part of the public api makes sense. Putting it in pipeline config seems odd. I guess the internal api section is the best place. The app/engine convenience methods should probably follow suit.